backend/Application/DTOs/Admin/AdminDTos.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.DTOs.Admin
+{
+    public class AdminDTos : IdentityUser
```

Comment:
Where is this used? If it's for retrieving data, we could just send the object directly, no need to map the objects, also for DTOs I think we should be using records instead of classes.